tests/test-blocks.php

```diff
@@ -0,0 +1,118 @@
+<?php
+/**
+ * Class Blocks Test
+ *
+ * @package Newspack_Listings
+ */
+
+use \Newspack_Listings\Newspack_Listings_Core as Core;
+use \Newspack_Listings\Utils as Utils;
+
+/**
+ * Blocks test case.
+ */
+class BlocksTest extends WP_UnitTestCase {
+	private static $listings           = []; // phpcs:ignore Squiz.Commenting.VariableComment.Missing
+	private static $default_attributes = []; // phpcs:ignore Squiz.Commenting.VariableComment.Missing
+
+	public function setUp() { // phpcs:ignore Squiz.Commenting.FunctionComment.Missing
+		// Remove any listings (from previous tests).
+		foreach ( self::$listings as $listing_id ) {
+			wp_delete_post( $listing_id );
+		}
+
+		// Set default block options for Curated List.
```

Comment:
Unfortunately not, because the test is calling `render_block` directly, so it's not pulling in the default attributes as if it were called as a callback from `register_block_type`. That said, the default attributes should still be pulled in where you suggest, in case we ever include any blocks that don't have a `view.php` (they currently all do, so this code isn't executed at all in this repo). Updated in 3201203.